apps/testing/src/main/java/com/akto/test_editor/Utils.java

```diff
@@ -617,4 +621,38 @@ public static String convertToHarPayload(String message, int akto_account_id, in
         return mapper.writeValueAsString(result);
     }
 
+    public static String extractValue(String keyValue, String key) {
+        String result = "";
+        if (keyValue.contains(key)) {
+            result = keyValue.split(key)[1].split("[,}]")[0];
+            result = result.replaceAll("\\}$", "");
+            result = result.trim();
+        }
+        return result;
+    }
+
+    public static ExecutorSingleOperationResp sendRequestToHostedServer(String requestUrl, String redirectUrl, String tokenVal){
```

Comment:
fix the naming of this method. how will someone reading this get to know what this does